doc-content/enterprise-only/cases/case-management-it-order-sample-proc.adoc

```diff
@@ -10,16 +10,21 @@ For more information about Red Hat Technology Preview support, see https://acces
 ====
 
 .Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
++
 
-Go to the *IT_Orders* sample project to view the predefined assets.
+{PRODUCT} provides a default space called *MySpace*, as shown in the following image. You can use the default space to create and test example projects.
++
 
-. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
-. Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Try Samples*.
+.Default space
+image::getting-started/spaces-teams.png[]
+
+. Click *Try Samples*.
 . Select *IT_Orders* and click *Ok*.
 
 The *Assets* view of the project opens. Select each example asset to explore how the project is designed to achieve the specified goal or workflow.
 
-Review the *orderhardware Business Processes (legacy)* asset, which will help you understand the business process flow.
+Review the *orderhardware Business Processes* asset, which will help you understand the business process flow.
```

Comment:
It's OK, but I would say.. asset to help you...

Reply:
Fixed.